ts/features/payments/onboarding/screens/PaymentsOnboardingSelectMethodScreen.tsx

```diff
@@ -85,7 +73,7 @@ const PaymentsOnboardingSelectMethodScreen = () => {
       <WalletOnboardingPaymentMethodsList
         isLoadingMethods={isLoadingPaymentMethods}
         onSelectPaymentMethod={handleSelectedPaymentMethod}
-        paymentMethods={availablePaymentMethods}
+        paymentMethods={availablePaymentMethods || []}
```

Comment:
```suggestion
        paymentMethods={availablePaymentMethods ?? []}
```
the outcome is the same, so it's not even close to a "functional" change, in fact I am pretty sure that in this situation it's just syntactic sugar.
That said, I find it "an easier read" if you will; what do you think?

Reply:
They are almost the same but `??` is more robust! In this case the left value could be `undefined`, so the nullish coalescing is better than the logical OR.
Changed in https://github.com/pagopa/io-app/pull/5680/commits/646ecd9d128deb2a4655066347b6e5c6d1105c2d